stories/templates/Checkbox/basic.js

```diff
@@ -3,16 +3,21 @@ import { html } from "lit-html";
 export const Template = args => {
   return html`
     <sgds-checkbox
-      .name=${args.name}
+      name=${args.name}
       ?disabled=${args.disabled}
       value=${args.value}
       ?required=${args.required}
       .checked=${args.checked}
       .invalid=${args.invalid}
       .valid=${args.valid}
-      .checkboxId=${args.checkboxId}
-      >${args.defaultSlot}<span slot="feedback">${args.feedbackSlot}<span></sgds-checkbox
+      ?invalidFeedback=${args.invalidFeedback}
```

Comment:
Aloha. is `invalidFeedback` a type: string?
```suggestion
      invalidFeedback=${args.invalidFeedback}
```

Reply:
thanks for the catch!